pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1166,7 +1166,23 @@ CREATE OR REPLACE VIEW SCHEMA_INFO.label AS
   FROM SCHEMA_CATALOG.label_key lk;
 
 
+CREATE OR REPLACE FUNCTION SCHEMA_CATALOG.delay_compression_job(ht_table name, new_start timestamptz) RETURNS VOID
+AS $$
+DECLARE
+  bgw_job_id int;
+BEGIN
+        SELECT job_id INTO bgw_job_id
+		FROM _timescaledb_config.bgw_policy_compress_chunks p
+		INNER JOIN _timescaledb_catalog.hypertable h ON (h.id = p.hypertable_id)
+		WHERE h.schema_name = 'SCHEMA_DATA' and h.table_name = ht_table;
 
+        --alter job schedule is not currently concurrency-safe (timescaledb issue #2165)
+        PERFORM pg_advisory_xact_lock(12377, bgw_job_id);
```

Comment:
Oh. How'd you get this number? Just a random one?

Reply:
yep random